README.md

```diff
@@ -30,7 +30,7 @@ Exquisite Corpse is a game invented by surrealist artists in the 1920's. Each pl
 
 ### Local Development
 
-Clone this repository and make it your current directory.
+Clone this repository and make it your current directory. In this root directory, create a local .env file by making a copy of .env.example named .env (e.g. `cp .env.example .env`.)
```

Comment:
We'll probably also want to tell them to do the same in the `server` directory.

Reply:
I did on line 48, but I'll try to make it more obvious